assets/script.js

```diff
@@ -18,4 +18,4 @@ function updateScreen() {
 	octo2.style.fill = hex;
 }
 
-window.onload = setInterval(function () { updateScreen(); }, 100);
+window.onload = setInterval(updateScreen, 1000);
```

Comment:
Are you sure about this 1s update interval?
Imagine the user loads the page at 0.999s: it will display 0s and a second later it shows 1s, while the time is 1.999s...